packages/experiments/src/components/Button/SplitButton/SplitButton.state.ts

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react';
+import { BaseState } from '../../../utilities/BaseState';
+import { ISplitButtonProps, ISplitButtonViewProps } from './SplitButton.types';
+
+export type ISplitButtonState = Pick<ISplitButtonViewProps, 'buttonRef'>;
+
+export class SplitButtonState extends BaseState<ISplitButtonProps, ISplitButtonViewProps, ISplitButtonState> {
```

Comment:
If core button has state, I'm curious why that can't be used for SplitButton. If SplitButton has two Buttons, don't each of those Buttons have their own state and ref?

Reply:
That could work. Let me try it.